docs/whats-new/dotnet-docs-mod2.md

```diff
@@ -1,115 +1,179 @@
+- [In-box BinaryFormatter implementation removed and always throws](../core/compatibility/serialization/9.0/binaryformatter-removal.md)
+- [SafeEvpPKeyHandle.DuplicateHandle up-refs the handle](../core/compatibility/cryptography/9.0/evp-pkey-handle.md)
+- [Some X509Certificate2 and X509Certificate constructors are obsolete](../core/compatibility/cryptography/9.0/x509-certificates.md)
 
 ## .NET fundamentals
 
 ### New articles
 
-- [Building native libraries](../core/deploying/native-aot/libraries.md)
-- [Install .NET Runtime with Snap](../core/install/linux-snap-runtime.md)
-- [Install .NET SDK with Snap](../core/install/linux-snap-sdk.md)
-- [Microsoft Testing Platform diagnostics overview](../core/testing/unit-testing-platform-diagnostics.md)
-- [MSTEST0025: Use 'Assert.Fail' instead of an always-failing assert](../core/testing/mstest-analyzers/mstest0025.md)
-- [MSTEST0026: Avoid conditional access in assertions](../core/testing/mstest-analyzers/mstest0026.md)
-- [NUnit runner overview](../core/testing/unit-testing-nunit-runner-intro.md)
-- [Persisted dynamic assemblies in .NET](../fundamentals/runtime-libraries/system-reflection-emit-persistedassemblybuilder.md)
-- [SYSLIB0054: Thread.VolatileRead and Thread.VolatileWrite are obsolete](../fundamentals/syslib-diagnostics/syslib0054.md)
-- [SYSLIB0055: AdvSimd.ShiftRightLogicalRoundedNarrowingSaturate\* methods with signed parameters are obsolete](../fundamentals/syslib-diagnostics/syslib0055.md)
+- [.NET Runtime metrics](../core/diagnostics/built-in-metrics-runtime.md)
+- [.NET SDK workload sets](../core/tools/dotnet-workload-sets.md)
+- [.NET uninstall tool overview](../core/additional-tools/uninstall-tool-overview.md)
+- [BinaryFormatter compatibility package](../standard/serialization/binaryformatter-migration-guide/compatibility-package.md)
+- [BinaryFormatter functionality reference](../standard/serialization/binaryformatter-migration-guide/functionality-reference.md)
+- [BinaryFormatter migration guide](../standard/serialization/binaryformatter-migration-guide/index.md)
+- [Choose a serializer](../standard/serialization/binaryformatter-migration-guide/choose-a-serializer.md)
+- [dotnet workload config](../core/tools/dotnet-workload-config.md)
+- [dotnet-core-uninstall dry-run](../core/additional-tools/uninstall-tool-cli-dry-run.md)
+- [dotnet-core-uninstall list](../core/additional-tools/uninstall-tool-cli-list.md)
+- [dotnet-core-uninstall remove](../core/additional-tools/uninstall-tool-cli-remove.md)
+- [Dynamic adaptation to application sizes (DATAS)](../standard/garbage-collection/datas.md)
+- [Example: Use OpenTelemetry with Azure Monitor and Application Insights](../core/diagnostics/observability-applicationinsights.md)
+- [Example: Use OpenTelemetry with OTLP and the standalone Aspire Dashboard](../core/diagnostics/observability-otlp-example.md)
+- [Example: Use OpenTelemetry with Prometheus, Grafana, and Jaeger](../core/diagnostics/observability-prgrja-example.md)
```

Comment:
These two articles are causing the build warnings. 

Did they move, or should these two be removed?

Reply:
It's because the files have upper-case characters in their names. I logged https://github.com/dotnet/docs-tools/issues/401.